control-plane/tests/test_cli_inspection.py

```
@@ -1456,6 +1456,31 @@ def test_service_admits_non_loopback_wazuh_ingest_from_trusted_proxy_peer(self)
         self.assertEqual(len(store.list(AlertRecord)), 1)
         self.assertEqual(len(store.list(AnalyticSignalRecord)), 1)
 
+    def test_service_admits_trusted_proxy_peer_with_surrounding_whitespace(self) -> None:
+        store, _ = make_store()
+        service = AegisOpsControlPlaneService(
+            RuntimeConfig(
+                host="0.0.0.0",
```

Ruff (0.15.9)

[error] Possible binding to all interfaces

(S104)